cpp/purify/yaml-parser.cc

```diff
@@ -15,6 +15,8 @@
 #include <yaml-cpp/yaml.h>
 #include "yaml-parser.h"
 #include <assert.h>
+#include <unistd.h>
```

Comment:
remove?

Reply:
Actually no story. Needed for the `getcwd` function

Reply:
Okay, that explains some other things. We should probably use boost for this, since it is a bit nicer. Look at https://theboostcpplibraries.com/boost.filesystem-files-and-directories